src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -88,15 +142,4 @@ public void hide() {
     public void focus() {
         getRoot().requestFocus();
     }
-
```

Comment:
Instead of removing the URL completely, I think it is better if we add a link to it at the bottom of the help text.